Libraries/RoslynTripleSlash/TripleSlashSyntaxRewriter.cs

```diff
@@ -88,6 +88,7 @@ public ...
     */
     internal class TripleSlashSyntaxRewriter : CSharpSyntaxRewriter
     {
+        private static readonly string[] ReservedKeywords = new[] { "abstract", "async", "await", "false", "null", "sealed", "static", "true", "virtual" };
```

Comment:
You could use Roslyn API to tell you if it is a reserved keyword: https://github.com/dotnet/roslyn/blob/4670412b9643db82b07e0e701d18e8b5c7f244ff/src/Compilers/CSharp/Portable/Syntax/SyntaxKindFacts.cs#L11

There are a bunch of static APIs there to tell you facts about the tokens.

Reply:
Didn't know that. Thanks for pointing it out.